src/NuGet.Clients/NuGet.CommandLine/Commands/PackCommand.cs

```diff
@@ -79,8 +79,10 @@ public ICollection<string> Exclude
         [Option(typeof(NuGetCommand), "CommandMSBuildPath")]
         public string MSBuildPath { get; set; }
 
-        // TODO: Temporarily hide the real ConfigFile parameter from the help text.
-        // When we fix #3230, we should remove this property.
+        [Option(typeof(NuGetCommand), "PackageCommandInstallPackageToOutputPath")]
+        public bool InstallPackageToOutputPath { get; set; }
```

Comment:
i don't see the OutputFileNamesWithoutVersion switch here.

Reply:
Sorry thought we wanted them all combined, will add another switch for the OutputFileNamesWithoutVersion.

Reply:
Fixed